sendbuffer.go

```diff
@@ -70,14 +71,14 @@ func (buf *sendBuffer) sendLoop(out chan []byte) {
 				select {
 				case <-windowAvailable:
 					// send allowed
-					out <- append(frame, buf.defaultHeader...)
+					w.Write(append(frame, buf.defaultHeader...))
```

Comment:
Return values are not checked in all callers, but I'm not sure if handling the error is any better than just ignoring it.